recipes/pydv/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "pydv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/pydv/-/archive/{{ version }}/pydv-{{ version }}.tar.gz
+  sha256: 6ef68ee9b1f921db7f21812eff4fb0c06ace602e2160dee3c814b6430450f3e1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - pydv = pydv.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gpstime
+    - matplotlib
+    - numpy
+    - python
+    - python-nds2-client
+
+test:
+  imports:
+    - pydv
+    - pydv.bufferdict
+    - pydv.const
+    - pydv.plotting
+    - pydv.util
+  commands:
+    - pydv --help
+
+about:
+  home: https://git.ligo.org/cds/pydv/
+  dev_url: https://git.ligo.org/cds/pydv/
+  license: GPLv3+
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```

---

@chrisburr, according to https://spdx.org/licenses/GPL-3.0.html the actual identifier should probably be `GPL-3.0-or-later`, I am happy to change it if the recommendation is to use SPDX identifiers in this field?

---

So far as I know the SPDX identifiers are prefered: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L60

(Thanks for the correction with `-or-later`, I should use that more often)